extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -75,9 +75,6 @@ class AmpInstagram extends AMP.BaseElement {
     /** @private {?Element} */
     this.iframe_ = null;
 
-    /** @private {?Promise} */
-    this.iframePromise_ = null;
```

Comment:
This is actually used in testing, so `@visibleForTesting`.

Reply:
Done.